src/api-client.ts

```diff
@@ -38,6 +40,7 @@ export class CortexApiClient {
       method: "POST",
       headers: {
         Authorization: `Bearer ${this.accessToken}`,
+        "User-Agent": `cortex-js-sdk-v${version}`,
```

Comment:
I would encourage us to user something closer to the spec (`product/version`): https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent

Maybe `cortex-nodejs-sdk/v0.0.9`? 

having the `/` also makes parsing a little bit easier in various contexts like sql/analytics.

Reply:
good thinking